learn/hoon/hoon-tutorial/generators.md

```diff
@@ -463,7 +463,7 @@ we can jump in with an example:
 =,  generators
 :-  %get
 |=  [* [url=tape ~] ~]
-^-  (sole-request:sole (cask json))
+^-  (sole-request (cask json))
```

Comment:
This is incorrect.  `sole` is not exposed here and this code fails with `find.sole-request`